migrate.py

```diff
@@ -256,12 +262,13 @@ def format_issue_body(issue, options):
 {content}
 
 {sep}
-- Bitbucket: https://bitbucket.org/{repo}/issue/{id}
+- Bitbucket: https://bitbucket.org/{owner}/{repo}/issue/{id}
```

Comment:
This is incorrect, as noted in the readme, the 'repo' specifies both the owner and the repo. This is necessary because the repoo 'owner' may not be the same as the bb username being used for authentication, especially if the owner is an organization.